packages/components/src/VirtualizedList/ListTable/ListTable.component.js

```diff
@@ -73,9 +73,7 @@ function ListTable(props) {
 			onRowClick={onRowClickCallback}
 			onRowDoubleClick={onRowDoubleClickCallback}
 			rowClassName={({ index }) =>
-				classNames(
-					...[theme['tc-list-item'], rowThemes, collection[index] && collection[index].className],
-				)
+				classNames(...['tc-list-item', rowThemes, collection[index] && collection[index].className])
```

Comment:
```suggestion
				classNames(...[theme['tc-list-headerRow'], rowThemes, collection[index] && collection[index].className])
```

Reply:
Problem is that this class is applied on each table item, not only header. 
That's what produced css rules application (like box-shadow).

Reply:
Then this might be the better place ton inject `theme['tc-list-headerRow']` ? 
https://github.com/Talend/ui/pull/3827/files#diff-97efde2bbcb9960078c7666f84dc0b8a3557b83349012cd539a1b2853cf828b1R115